Formula/swift.rb

```diff
@@ -34,7 +34,7 @@ class Swift < Formula
   # This is mostly community sourced, so may be not necessarily be accurate.
   depends_on xcode: ["13.0", :build]
 
-  depends_on "python@3.10"
+  depends_on "python@3.11"
 
   # HACK: this should not be a test dependency but is due to a limitation with fails_with
   uses_from_macos "llvm" => [:build, :test]
```

Comment:
Should be able to remove hack, but probably not worth doing it right now when we are trying to finish this to ship `icu4c` update.